src/DefaultJsonResponseNegotiator.cs

```diff
@@ -19,7 +27,7 @@ public bool CanHandle(IList<MediaTypeHeaderValue> accept)
         public async Task Handle(HttpRequest req, HttpResponse res, object model, CancellationToken cancellationToken)
         {
             res.ContentType = "application/json; charset=utf-8";
-            await res.WriteAsync(JsonConvert.SerializeObject(model), cancellationToken);
+            await res.WriteAsync(JsonConvert.SerializeObject(model, new JsonSerializerSettings { ContractResolver = _contractResolver }), cancellationToken);
```

Comment:
move the settings to the ctor and hardcode the camelcase resolver

Reply:
JsonSerializerSettings doesn't have an interface...

Reply:
Sorry I meant the body if it
``` 
        public DefaultJsonResponseNegotiator()
         {
           this.jsonSettings = new JsonSerialzierSettings{ContractResolver = new CamelCasePropertyNamesContractResolver() };
         }
```